### PR TITLE
Financing budgeting/fin 29 receivables

### PR DESCRIPTION
**Pull request by tatul-m**
Please ignore all issues which same as in Payables.
Waiting for their resolving to merge

### CODE REVIEW

**Review by igortokhunts — APPROVED** [review]

File: src/src_V2/modules/manager/financial/components/receivables/Receivables.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      receivable.remaining = remainingCost(receivable.totalAmount, receivable.paidAmount);
+      return receivable;
+    });
+    setReceivables(processedReceivables);
+  }
+
+  const filterReceivables = useCallback(() => {
+    // filtering receivables by Tour Name/ID/Partner
+    const filteredReceivables = receivables.filter(
+      (item) =>
```
Comment (TigranYengiba04) [inline src/src_V2/modules/manager/financial/components/receivables/Receivables.jsx:41]: I think this approach will be more readable: `({ tourName, receivableId, partner }) => [tourName, receivableId, partner].some(item => item.toLowerCase().includes(searchTxt))`

File: src/src_V2/modules/manager/financial/components/receivables/Receivables.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    );
+    setFilteredReceivables(filteredReceivables);
+  });
+
+  useEffect(getReceivables, []);
+
+  useEffect(filterReceivables, [searchTxt, receivables]);
+
+  return (
+    <>
```
Comment (TigranYengiba04) [inline src/src_V2/modules/manager/financial/components/receivables/Receivables.jsx:54]: pls remove fragment

File: src/src_V2/modules/manager/financial/components/receivables/edit/drawer-content/notes/Notes.jsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          value={values.notes}
+          onBlur={handleBlur}
+          onChange={handleChange}
+          error={touched.notes && errors.notes}
+          helperText={touched.notes && errors.notes}
+          InputProps={{
+            endAdornment: (
+              <>
+                <Box className={styles.lengthSection}>
+                  <p>{!!values?.notes?.length && `${values.notes.length}/500 letter`}</p>
```
Comment (TigranYengiba04) [inline src/src_V2/modules/manager/financial/components/receivables/edit/drawer-content/notes/Notes.jsx:59]: pls use Typography

File: src/src_V2/modules/manager/financial/components/receivables/edit/drawer/EditDrawer.jsx [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+  const anchor = 'right';
+  const [unsavedPopupOpened, setUnsavedPopupOpened] = useState(false);
+  const [applyPopupOpened, setApplyPopupOpened] = useState(false);
+
+  const rowEditForm = useFormik({
+    validationSchema: receivablesRowListSchema,
+    initialValues: receivablesRowListInitialValues(clickedRow),
+  });
+
+  const toggleDrawer = (isOpened) => {
```
Comment (TigranYengiba04) [inline src/src_V2/modules/manager/financial/components/receivables/edit/drawer/EditDrawer.jsx:36]: pls remove isOpened argument and get it from drawerState

**Review by TigranYengiba04 — CHANGES_REQUESTED** [review]

**Pull request merged**